tests/cbmc/proofs/Makefile.common

```diff
@@ -0,0 +1,279 @@
+CBMCFLAGS += --nan-check
+CBMCFLAGS += --pointer-check
+CBMCFLAGS += --pointer-overflow-check
+CBMCFLAGS += --signed-overflow-check
+CBMCFLAGS += --undefined-shift-check
+CBMCFLAGS += --unsigned-overflow-check
+CBMCFLAGS += --unwind 1
+CBMCFLAGS += --unwinding-assertions
+
+################################################################
+# Setup the unwindset
+# TODO: CBMC should accept multiple unwindset flags, making the substutions here
+# unnecessary.
+# Tracked in https://github.com/diffblue/cbmc/issues/5185
+ifneq ($(UNWINDSET),)
+CBMC_UNWINDSET := --unwindset $(subst $(SPACE),$(COMMA),$(strip $(UNWINDSET)))
+endif
+CBMCFLAGS += $(CBMC_UNWINDSET)
+
+################################################################
+# Set C compiler defines
+CBMC_OBJECT_BITS ?= 8
+CBMCFLAGS +=  --object-bits $(CBMC_OBJECT_BITS)
+DEFINES += -DCBMC_OBJECT_BITS=$(CBMC_OBJECT_BITS)
+DEFINES += -DCBMC=1
+
+################################################################
+# Setup include directory order
+# HELPERDIR must go before any other includes to override default
+# implementations by stubs
+INC += -I$(HELPERDIR)/include/
+INC += -I$(SRCDIR)
+INC += -I$(SRCDIR)/api
+
+################################################################
+# Enables costly checks (e.g. ones that contain loops)
+# Don't execute deep checks by default
+AWS_DEEP_CHECKS ?= 0
+DEFINES += -DAWS_DEEP_CHECKS=$(AWS_DEEP_CHECKS)
+
+################################################################
+# Remove function bodies that are not used in the proofs. 
+# Removing the function from the goto program helps CBMC's
+# function pointer analysis
+
+# We override abort() to be assert(0)
+ABSTRACTIONS += $(HELPERDIR)/stubs/abort_override_assert_false.c
+REMOVE_FUNCTION_BODY += --remove-function-body abort
+
+# Users can remove additonal functions using this paramater
+REMOVE_FUNCTION_BODY += $(ADDITIONAL_REMOVE_FUNCTION_BODY)
+
+################################################################
+# Determine the names of the goto files to build
+#DEPENDENT_GOTOS = $(call GOTO_FROM_C,$(DEPENDENCIES))
+ENTRY_GOTO = $(GOTODIR)/$(ENTRY)
+
+################################################################
+# Temporarily disable UNWIND_GOTO, SIMPLIFY until the feature is
+# stable
+UNWIND_GOTO ?= 0
+SIMPLIFY ?= 0
+
+################################################################
+# Build targets that make the relevant .goto files
+
+# Other libraries that use this Makefile template may wish to do
+# actual work here
+setup_dependencies:
+	echo "Setting up dependencies"
+
+# Here, whenever there is a change in any of ANSI-C source
+# dependency files, make will take action. However, to make
+# sure changes in the headers files will also trigger make,
+# the user must run make clean first.
+$(ENTRY_GOTO)0.goto: setup_dependencies $(ENTRY).c $(DEPENDENCIES)
+	mkdir -p $(dir $@)
+	mkdir -p $(dir $(call LOG_FROM_ENTRY,$@))
+	$(call DO_GOTO_CC,\
+		--export-function-local-symbols \
+			$(CBMC_VERBOSITY) \
```

Comment:
Suggest a single tab at the beginning of the line, followed by enough spaces to align the text

Reply:
Fixed